src/main/java/net/sf/rails/ui/swing/StatusWindow.java

```diff
@@ -527,12 +528,61 @@ public void updateStatus(boolean myTurn) {
             gameStatus.setPriorityPlayer(gameUIManager.getPriorityPlayer().getIndex());
         }
 
+        // Must Special menu be enabled?
+        boolean enabled = updateSpecialActionMenu();
+        specialMenu.setOpaque(enabled);
+        specialMenu.setEnabled(enabled);
+        specialMenu.repaint();
+
+        passButton.setEnabled(false);
```

Comment:
Should these really be here? Do you remember why you added this here as we have logic triggered via `GameUIManager` to handle the refresh/state of pass/autoPass

Reply:
Actually the buttons were already disabled above at line 485 so I'm assuming this is a leftover from merging?

Reply:
Old Code most likely, need to check again